plugin-server/src/main/ingestion-queues/session-recording/process-event.ts

```diff
@@ -45,10 +45,64 @@ export interface SummarizedSessionRecordingEvent {
     snapshot_source: string | null
 }
 
+// this is of course way more complicated than you'd expect
```

Comment:
consider the mobile ones as well https://github.com/PostHog/posthog-android/blob/d57bc993755327482163d8fc1f7e2aa606c70cdd/posthog-android/src/main/java/com/posthog/android/replay/internal/LogcatParser.java#L56-L68

Reply:
it's going to take so long to get used to not only considering browser APIs 🤣 

luckily they overlap with the browser ones - phew!